raiden/tests/conftest.py

```diff
@@ -20,6 +20,15 @@
 from raiden.tests.utils.transport import make_requests_insecure
 from raiden.utils.cli import LogLevelConfigType
 
+pytest.register_assert_rewrite('raiden.tests.utils.eth_node')
```

Comment:
these are the modules inside `raiden.tests.utils` which use `assert`

Reply:
I went through the documentation for this here: https://docs.pytest.org/en/latest/reference.html#pytest-register-assert-rewrite but i couldn't tell what exactly this does. Can you explain the benefit of doing this?

Reply:
Pytest does runtime opcode rewriting. That is how we get these nice reports on failed asserts:

```
>        assert a == b
E       assert 1 == 2
```
The important thing here is that it shows the value of `a` and `b`. But this is only done for the test modules, imported modules are not rewritten and the test report is like this:

```
>       assert a == b
E       AssertionError
```

This `register_assert_rewrite` allows opcode rewriting for additional modules. I'm using it to get better error reports for our assert utilities

Reply:
Here is an example to show opcode rewriting and the difference in the output: https://gist.github.com/hackaugusto/6cb9c9117c835b9a19e958e8022107bc

Reply:
Also, I'm doing this because at the moment I can not tell why this test is failing: https://circleci.com/gh/raiden-network/raiden/40429#tests/containers/2

Reply:
That's interesting. Didn't know you can also affect asserts for imported modules so that they show the values like this. Only problem I can see with this method is that this list of files which use assert can change in the future and we probably will forget changing it here. But I guess that's minor.